ios/FluentUI/Notification/NotificationTokenSet.swift

```diff
@@ -277,44 +256,22 @@ public class NotificationTokenSet: ControlTokenSet<NotificationTokenSet.Tokens>
             case .outlineWidth:
                 return .float { theme.globalTokens.borderSize[.thin] }
 
-            case .ambientShadowColor:
-                return .dynamicColor {
-                    switch style().isToast {
-                    case true:
-                        return theme.aliasTokens.shadow[.shadow16].colorOne
-                    case false:
-                        return DynamicColor(light: ColorValue.clear)
-                    }
-                }
-
-            case .ambientShadowBlur:
-                return .float { theme.aliasTokens.shadow[.shadow16].blurOne }
-
-            case .ambientShadowOffsetX:
-                return .float { theme.aliasTokens.shadow[.shadow16].xOne }
-
-            case .ambientShadowOffsetY:
-                return .float { theme.aliasTokens.shadow[.shadow16].yOne }
-
-            case .perimeterShadowColor:
-                return .dynamicColor {
-                    switch style().isToast {
-                    case true:
-                        return theme.aliasTokens.shadow[.shadow16].colorTwo
-                    case false:
-                        return DynamicColor(light: ColorValue.clear)
+            case .shadow:
+                return .shadowInfo {
+                    if style().isToast {
+                        return theme.aliasTokens.shadow[.shadow16]
+                    } else {
+                        return ShadowInfo(colorOne: DynamicColor(light: ColorValue.clear),
```

Comment:
Would it be worth adding this "empty" shadow in AliasTokens in case any other control might need it?

Reply:
Great idea

Reply:
Should we just conditionally apply the shadow to the notification, like we do with the button?